Mobile menu modal

jaimevillegas opened the pull request:
We have made the following modifications to the project:

- Modified HTML file adding a new `<div>` with the contents of modal window.
- Modified CSS file adding styling properties applied to the modal window.
- Created JS file in order to add functionality to the modal window.

Thank you a lot for taking the time to review our code and suggest any changes you consider.

BenMukebo commented on style.css:
```diff
@@ -24,7 +24,83 @@ a {
   justify-content: space-between;
   align-items: stretch;
-  border: 1px solid red;
+
+  /* border: 1px solid red; */
+}
+
+#menu-toggle {
+  background: none;
+  margin: 0;
+  padding: 0;
+
+  /* boder: 1px solid red; */
+}
+
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+
+  /* height: 100vh; */
+
+  /* overflow: auto; */
+  background-color: rgb(0 0 0);
+  background-color: rgb(0 0 0 / 40%);
+}
+
+.nav-menu-content {
+  background-color: #fefefe;
+
+  /* margin: 15% auto; */
```
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

BenMukebo commented on style.css:
```diff
@@ -1,13 +1,13 @@
   margin: 0;
   padding: 0;
-  max-width 375px;
+  max-width: 375px;
```
- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇

BenMukebo reviewed (changes requested):
### Hi @TEAM,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository have been set up correctly. ✔️  
-  When the user clicks the hamburger button, the mobile menu appears.✔️ 
- When the user clicks the close (X) button, the mobile menu disappears.✔️ 
- When the user clicks any of the mobile menu options, a correct part is displayed and the mobile menu disappears 👏🏻 
- You have followed  JavaScript best practices. ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```
- [ ] Please remove all unnecessary titles since they are empty  or add details to them (Prerequisites, run tests Deployment ....) 🚮 


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jaimevillegas commented:
Hello @BenMukebo! Thank you so much for your code review.

I have worked on the suggestions that you have made:

- Updated `style.css` removing unnecessary code.
- Updated `README.md` removing blank titles. 

Thank you for your advice!

ShadyShawkat reviewed (approved):
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

![Image](https://media.giphy.com/media/jJQC2puVZpTMO4vUs0/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
